cards.js

```diff
@@ -17,7 +17,46 @@ function shuffle(array){
 }
 
 function buildDeck(){
+  var deck = [];
+    for (i = 0; i < values.length; i++){
+      newCard = {suit: "Clubs", value: values[i]};
+      deck.push(newCard);
+      }
+    for (i = 0; i < values.length; i++){
+      newCard = {suit: "Diamonds", value: values[i]};
+      deck.push(newCard);
+        }
+    for (i = 0; i < values.length; i++){
+      newCard = {suit: "Hearts", value: values[i]};
+      deck.push(newCard);
+        }
+    for (i = 0; i < values.length; i++){
+      newCard = {suit: "Spades", value: values[i]};
+      deck.push(newCard);
+        }
+  return deck
+}
+
+function highCard(){
+  shuffledDeck = shuffle(buildDeck());
+  var playerOne = shuffledDeck[0];
+  var playerTwo = shuffledDeck[1];
+  console.log("Player 1 has the " + playerOne["value"] + " of " + playerOne["suit"] + "!");
+  console.log("Player 2 has the " + playerTwo["value"] + " of " + playerTwo["suit"] + "!");
+
+  var cardValueOrder = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"]
 
-// Put some code in here!
+  if (cardValueOrder.indexOf(playerOne["value"]) > cardValueOrder.indexOf(playerTwo["value"])){
```

Comment:
clever way to solve the problem! It was a bit hard to follow though, so you might want to see if you can find a clearer way to do this... (e.g. see some of the other solutions, like this one: https://github.com/ga-dc/game-of-war/pull/23/files)